### ISSUE TITLE
Resize, when 200% Zoomed Items present in Home screen are getting truncated: A11y_.NET Core_Recipes App Home Screen_Home Screen_Resize.

### ISSUE THREAD

**Issue by keerthiduvvuri** [dotnet/maui#23005]
### GIT Hub Tags:
#AI Limited; #A11yTCS; #A11ySev2; #.NET Core; #E2E-.NET Core-Android-MAUI-JUNE2024; #Talkback; #Android; #WCAG1.4.4; #Resize Text; #FTP; #A11yMAS;

### Environment and OS details:
IOS Version 17

### Reproduction Steps

1. Install and open ".NET MAUI" app.
2. "Home Screen" will open.
3. Observe that Resize, when 200% Zoomed Items present in Home screen are getting truncated

### Actual behavior:
Resize, when 200% Zoomed Items present in Home screen are getting truncated.

### Expected behavior:
Resize, when 200% Zoomed Items present in Home screen should not be getting truncated.

### User Impact
Visually impaired users will face difficulty if Resize, when 200% Zoomed Items present in Home screen are getting truncated.

### Version with bug

9.0.0-preview.4.10690

### Is this a regression from previous behavior?

Not sure, did not test other versions

### Last version that worked well

9.0.0-preview.4.10690

### Affected platforms

iOS

**Comment by ninachen03** [comment]
Verified this issue on VS Code 1.90.0 (8.0.70-ci.net8.24312.3 & 8.0.40) Can repro on iOS platform.

**Comment by Yash14j** [comment]
GithubTags:#Rev:yaja;

**Comment by tj-devel709** [comment]
@keerthiduvvuri could you explain what the resizing is exactly? I tried zooming in with the iOS Accessibility Zoom in the settings app, but it didn't look like the photos you posted. Maybe I am missing a way to do the resizing. Thanks!

**Comment by keerthiduvvuri** [comment]
Hi @tj-devel709, In IOS Phones there is zooming text, where we can zoom the application to maximum level, so that users with low vision people can zoom the application and access it as per their convince.
Steps to follow in IOS:
Settings -> Accessibility -> Display & Text size -> Large Text.
Could you please drag to Large Text need to be increased to maximum level.
Once it is increased the application will be zoomed and you can find the issue where it is getting truncated.

#Regressed:07-11-2024;
